src/test/ui/no-implicit-prelude-nested.stderr

```diff
@@ -3,50 +3,24 @@ error[E0405]: cannot find trait `Add` in this scope
    |
 LL |         impl Add for Test {}
    |              ^^^ not found in this scope
-   |
-help: consider importing this trait
-   |
-LL |         use std::ops::Add;
-   |
```

Comment:
Isn't this a regression?

Reply:
The regression is only for edition 2015. But for 2018, it is better, because `use ::std::ops:Add` should have been suggested instead, as `use std::ops:Add` inside `baz` causes an error - ```use of undeclared type or module `std` ```.

Reply:
I push a rewrite that addresses this.